core/src/main/java/hudson/cli/CliTransportAuthenticator.java

```diff
@@ -1,40 +0,0 @@
-package hudson.cli;
-
-import hudson.ExtensionList;
-import hudson.ExtensionPoint;
-import hudson.remoting.Channel;
-
-/**
- * @deprecated No longer used.
- */
-@Deprecated
-public abstract class CliTransportAuthenticator implements ExtensionPoint {
```

Comment:
Fine, this is completely unused. (Formerly used only by a module, but we have removed that reference and released a new version of the module and bundled it.)

Reply:
I think I declined to remove this in #3838 due to timing issues, FYI.

Reply:
Timing issues of code execution? Or of removing it at the same time as the others? 

My intent here was to see if we could accept the removal now that more time has elapsed. Maybe we should just wait on both of these.

Reply:
> of removing it at the same time as the others

This, because I think there was still an outstanding reference at the time. Now it should be fine to remove.